template/package.json

```diff
@@ -7,8 +7,8 @@
     "node": ">=10.0.0"
   },
   "scripts": {
-    "lint": "eslint . && npm run lint:lockfile",
-    "lint:lockfile": "lockfile-lint --path package-lock.json --type npm --validate-https --allowed-hosts npm yarn",
+    "lint": "eslint . && <%= npmClient %> run lint:lockfile",
+    "lint:lockfile": "lockfile-lint --path package-lock.json --type <%= npmClient %> --validate-https --allowed-hosts npm yarn",
```

Comment:
so here there is a `package-lock.json` which is for npm, and we need to make it either that, or `yarn.lock` if the user chooses yarn so we'll need a conditional here.